ui/grpc-web/.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "third_party/grpc"]
```

Comment:
can we avoid using .git submodules? They can complicate a build process.

Reply:
grpc-web folder will be removed